.github/workflows/swift-toolchain.yml

```diff
@@ -1446,7 +1476,10 @@ jobs:
                 -D LIBXML2_WITH_PYTHON=NO `
                 -D LIBXML2_WITH_TESTS=NO `
                 -D LIBXML2_WITH_THREADS=YES `
-                -D LIBXML2_WITH_ZLIB=NO
+                -D LIBXML2_WITH_ZLIB=NO `
+                -D CMAKE_POSITION_INDEPENDENT_CODE=${{ matrix.os == 'Android' && 'YES' || 'NO' }} `
```

Comment:
I think that we can set this unconditionally always, can't we?

Reply:
Done.